### ISSUE TITLE
Handle ghost detections (at DB & pkg)

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#68]
Here's a quick overview of the data that will be included in the dataset we will publish. @PieterjanVerhelst @IPauwels can you have a look if this makes sense? Let me know if you need more info.

animal_project_name | animals.scientific_name | detections | individuals | stations | start_date | end_date
--- | --- | --- | --- | --- | --- | ---
2011 Rivierprik | Lampetra fluviatilis | 114605 | 29 | 29 | 2011-12-14 | 2012-07-03
2012 Leopoldkanaal | Anguilla anguilla | 2215829 | 92 | 60 | 2012-07-04 | 2017-03-12
2014 Demer | Petromyzon marinus | 42 | 1 | 1 | 2015-05-06 | 2015-05-12
2014 Demer | Rutilus rutilus | 11030 | 2 | 9 | 2014-04-19 | 2014-06-28
2014 Demer | Silurus glanis | 86023 | 9 | 46 | 2014-04-25 | 2018-01-31
2014 Demer | Squalius cephalus | 139013 | 2 | 10 | 2014-04-30 | 2015-02-12
2015 Dijle | Anguilla anguilla | 41798 | 1 | 7 | 2015-05-01 | 2015-10-15
2015 Dijle | Cyprinus carpio | 4944 | 2 | 9 | 2015-04-23 | 2015-11-06
2015 Dijle | Platichthys flesus | 101488 | 8 | 28 | 2015-04-29 | 2016-04-08
2015 Dijle | Rutilus rutilus | 7870 | 4 | 9 | 2015-04-23 | 2015-09-14
2015 Dijle | Silurus glanis | 78331 | 11 | 25 | 2015-04-22 | 2017-09-16

**Comment by peterdesmet** [comment]
@PieterjanVerhelst I don't know. But I would like you to have a look at the original question, which is now at https://github.com/inbo/etn-occurrences/issues/7 😄

**Comment by peterdesmet** [comment]
@PieterjanVerhelst @jreubens @IPauwels this issue has been dormant. No need to read it all, the basic question is: do you want an automatic assessment (by the database (ideally) or package) to assess which detections are likely to be ghost detections?

**Comment by PieterjanVerhelst** [comment]
I would say yes, to some degree. That is, all 'detections' occurring after the battery of the transmitter dropped dead, I would consider ghost detections. Other forms of uncertainty  are up to the researcher to decide what to do with it (include it in analysis or not). However, I think @jreubens may have some additional thoughts on this one ;-).

**Comment by IPauwels** [comment]
I will leave it up to Jan as well, but had this small thought: you are never sure about the actual battery-end-date of the transmitter isn't it? So perhaps detections after the expected end date are still real detections.. Or did I understand your suggestion wrongly PJ?

**Comment by PieterjanVerhelst** [comment]
Transmitters have an end date and on that particular date, the transmitter runs dead.

**Comment by jreubens** [comment]
Yes this is needed! However to my opinion this should be tackled on DB level.

**Comment by peterdesmet** [comment]
@jreubens should I leave this issue open then? Or are you following this up on your side?

**Comment by PietrH** [comment]
@jreubens , @peterdesmet Can this be closed?
